scripts/mcp-monitor.sh

```
@@ -0,0 +1,424 @@
+    else
+        log_info "MCP monitor daemon is not running"
+        return 1
+    fi
+}
+
+# Function to rotate logs if they get too large
+rotate_logs_if_needed() {
+    local max_size_kb=1024  # 1MB limit
+    
+    # Rotate main log file
+    if [ -f "$LOG_FILE" ] && [ $(du -k "$LOG_FILE" 2>/dev/null | cut -f1) -gt $max_size_kb ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)